app/services/segment_delete_service.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+class SegmentDeleteService
+  SEGMENT_URI = URI('https://gdpr.segment.com/graphql')
```

Comment:
It should be configurable IMO.

Reply:
really? @guicassolato What for? to use other services? for this service this is the only possible link that can work, and if it is for other services (not only segment) then the whole code should adapt to different services 😕 imo that's not ideal... it is adding a loooot of unnecessary complexity...

Reply:
So if URL changes we (and customers) don't need to change the code, rebuild the image and redeploy. It's an external service. It should not be hardcoded.

Reply:
@guicassolato If the URL changes then the whole API can change, but they still should provide compatibility with this old version then.
But letting customers to add here any possible uri in the world seems to me like asking for problems... 😱 I will do it but I will do it in a separate commit because it feels like a very scary change 😱

Reply:
I still think this UTL should be configurable.